core/src/main/java/org/apache/accumulo/core/manager/balancer/TabletStatisticsImpl.java

```diff
@@ -46,7 +46,7 @@ public long getNumEntries() {
   }
 
   @Override
-  @SuppressWarnings("removal")
+  @Deprecated
```

Comment:
Is there a reason why this is `@Deprecated` vs `@SuppressWarnings("deprecated")`

Reply:
Chatted with @ctubbsii  and responding here to resolve this comment. 
So this is a method that uses an interface which is deprecated. We need to also deprecate the method since removal of the interface would still allow this method to be referenced without errors.